themes/loop/css/styles.css

```diff
@@ -1217,6 +1222,7 @@ label {
     -ms-box-sizing: border-box;
     box-sizing: border-box;
     word-wrap: break-word;
+    *behavior: url(url("../lib/boxsizing.htc"));
```

Comment:
It's should not be inside a url, but the issue is in the compilation of sass to css and the current setup for the theme don't work in node 4.x, 5.x of 6.x... So this is outside the scope of this PR.

If it needs to be fixed I think the way to go is creating an support ticket to loop on getting this fixed... I would expect this to take from 1-2 hours as some of the sass may need to be re-written to use the current versions.